ext/opcache/jit/zend_jit_ir.c

```diff
@@ -17195,7 +17195,9 @@ static void jit_frameless_icall1(zend_jit_ctx *jit, const zend_op *opline, uint3
 	ir_ref op1_ref = jit_ZVAL_ADDR(jit, op1_addr);
 	jit_set_Z_TYPE_INFO(jit, res_addr, IS_NULL);
 	if (opline->op1_type == IS_CV && (op1_info & MAY_BE_UNDEF)) {
-		zend_jit_zval_check_undef(jit, op1_ref, opline->op1.var, opline, 1);
+		op1_ref = zend_jit_zval_check_undef(jit, op1_ref, opline->op1.var, opline, 1);
+		op1_info |= MAY_BE_NULL;
```

Comment:
It may make sense to reset ``MAY_BE_UNDEF`` flag, but most probably this won't make any difference.

Reply:
Right of course, I'll update this and when CI passes merge this.